generated/provider_dependencies.json

```diff
@@ -107,6 +107,12 @@
       "vertica"
     ]
   },
+  "apache.impala": {
+    "deps": [
+      "impyla>=0.18.0,<1.0"
```

Comment:
Why should we limit?
This invites questions when 1.0 will be released so if there is no good reason to limit in advanced we should relax the dependency.

Reply:
I believe it could broke some dependency as exactly what happened on old PR and we needed to wait for new version of `impyla`